src/VisualStudio/Core/Def/Implementation/AbstractEditorFactory.cs

```diff
@@ -201,13 +200,6 @@ int IVsEditorFactory.Close()
                         return null;
                     }
```

Comment:
The code that retrieves the `IVsShell` here can be removed as well. We should move all of that into the VS designer code.

Reply:
Ok, will do :)

Reply:
Whoops, a little too aggressive removing that try catch, will fix.